clients/privacy-center/features/consent/GpcMessages.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Badge,
+  Box,
+  Link,
+  Stack,
+  Text,
+  HStack,
+  WarningTwoIcon,
```

Comment:
This is the icon that comes from Chakra, not our warning icon that has a hard-coded yellow circle. This one is closer to what the designs want, but still isn't quite right.  https://github.com/ethyca/fides/issues/2621

Reply:
Thanks for creating this issue.